spaces/domains.json

```
@@ -1,4 +1,5 @@
 {
+  "governance.cyberestate.io: "cyberrealestate.eth",
```

You are missing a quote.
```suggestion
  "governance.cyberestate.io": "cyberrealestate.eth",
```